core/storage/base_model/gae_models.py

```diff
@@ -1299,24 +1342,11 @@ def commit(
                 raise Exception(
                     'Invalid change list command: %s' % commit_cmd['cmd'])
 
-        commit_type = (
-            feconf.COMMIT_TYPE_CREATE
-            if self.version == 0
-            else feconf.COMMIT_TYPE_EDIT
+        models_to_put_values = self.get_models_to_put_values(
```

Comment:
Can we just call this models_to_put? It makes more sense that way. The current name sounds like you're "putting values into something".